spec/requests/zones_spec.rb

```diff
@@ -25,6 +25,42 @@
   end
 
   context "edit", :edit do
+    it "can create zone authentication" do
+      api_basic_authorize action_identifier(:zones, :edit)
+
+      zone   = FactoryBot.create(:zone, :description => "Current Zone description")
+      params = gen_request(:edit,
+                           "description"               => "Updated Zone Description",
+                           "authentication_attributes" => {"userid" => "foo", "password" => "bar"})
+
+      expect(zone.authentication).to eq(nil) # sanity check
```

Comment:
Shouldn't this be `zone.authentications` since it's a `has_many`?

Reply:
@bdunne no, since this is meant to pull in the changes from here:

https://github.com/ManageIQ/manageiq/pull/21047